src/openforms/registrations/contrib/email/templates/emails/email_registration.txt

```diff
@@ -9,8 +9,14 @@ Our reference: {{ public_reference }}
 Payment order ID: {{payment_order_id}}
 {% endblocktrans %}
 {% else %}{% blocktrans trimmed %}
-Submission details for {{form_name}} (submitted on {{datetime}})
+Submission details for {{ form_name }} (submitted on {{ datetime }})
 {% endblocktrans %}
 {% endif %}
-{% for key, value in submitted_data %}
-- {{ key }}: {% display_value value %}{% endfor %}{% endautoescape %}
+
+{% for submission_step_node in renderer.get_children %}
+{{ submission_step_node.render }}
+
+{% for component_node in submission_step_node.get_children %}
+{% whitespace component_node.depth base=' ' %}- {{ component_node.label }}{% if not component_node.is_layout %}:{% endif %} {{ component_node.display_value }}{% endfor %}
```

Comment:
Also here check for empty label/value like in html template?

Reply:
Updated the PR #1571 for that!